client/js/components/pages/HomePage.js

```diff
@@ -1,348 +1,32 @@
 import React, { Component } from 'react';
+import getGoogleApiClient from 'google-client-api';
 
-class HomePage extends Component {
-  constructor() {
-    super();
-    this.state = { visibility: false };
-  }
+import Logo from '../../../img/xteam-vertical.png';
+import 'file?name=[name].[ext]!../../../img/xteam-vertical.png';
 
-  hidePanel() {
-    this.setState({ visibility: true });
+class HomePage extends Component {
+  componentDidMount() {
+    getGoogleApiClient(gapi => {
+      gapi.load('signin2', () => {
+        gapi.signin2.render('g-signin2', {
+          longtitle: true,
+          width: 250,
+          height: 50
+        });
+      });
+    });
   }
 
   render() {
-    const visibility = this.state.visibility ? 'hidden' : '';
     return (
```

Comment:
maybe we don't need the homepage at all. what I tried to do is the "homepage screen" that appears before login, after login we don't need that page anymore, the "inside-app" default view can be the map only. if you want to make this with better approach, go for it. would be good idea if we first agree on https://github.com/x-team/x-map/issues/57 about this.